types/networks/Cargo.toml

```diff
@@ -15,4 +15,5 @@ serde_json = "1.0"
 mainnet = []
 interopnet = []
 devnet = []
-conformance = []
\ No newline at end of file
+conformance = []
+calibnet = []
```

Comment:
You can configure your IDE to automatically insert new lines at the end of the file.